libs/interpret-vision/src/lib/VisionExplanationDashboard/VisionExplanationDashboard.tsx

```diff
@@ -172,7 +211,8 @@ export class VisionExplanationDashboard extends React.Component<
         image,
         index,
         predictedY: dataSummary.predictedY[index],
-        trueY: dataSummary.trueY[index]
+        trueY: dataSummary.trueY[index],
+        [this.state.otherMetadataFieldName]: 50
```

Comment:
why is this value here hard-coded to 50?

Reply:
shouldn't this come from test data or python backend?

Reply:
This will be overwritten by the model assessment PR where it connects to the backend